app/containers/Header/components/HeaderButton/__snapshots__/HeaderButtons.test.tsx.snap

```diff
@@ -568,11 +568,11 @@ exports[`Story Snapshots: Badge should match snapshot 1`] = `
       }
     >
       <Text
-        accessibilityRole="header"
         aria-level="1"
         collapsable={false}
         numberOfLines={1}
         onLayout={[Function]}
+        role="heading"
```

Comment:
Unintentional change, but we're good apparently https://reactnative.dev/docs/text#role